core/sds-aspect-model-document-generators/src/main/java/io/openmanufacturing/sds/aspectmodel/generator/json/AspectModelJsonPayloadGenerator.java

```diff
@@ -82,6 +82,8 @@
 import io.openmanufacturing.sds.metamodel.datatypes.Curie;
 import io.openmanufacturing.sds.metamodel.impl.BoundDefinition;
 import io.openmanufacturing.sds.metamodel.loader.AspectModelLoader;
+import io.openmanufacturing.sds.metamodel.loader.Instantiator;
```

Comment:
Are these imports really used?

Reply:
No, this is a leftover. I removed all unused imports from the project now.